src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static User defaultUser;
+    private static User user;
+    private static RegistrationService service;
+    private static int counter = 0;
+    private static int expectedSize = 0;
+
+    @BeforeAll
+    static void beforeAll() {
+        defaultUser = new User();
+        defaultUser.setId(1L);
```

Comment:
I think it's necessary, because below you register you default user and in that method he will give his ID